db.sql.visualeditor/external/javacc-3.2-license.txt

```diff
@@ -4,6 +4,7 @@ Description: Parser generator for SQL parser.
 Origin: https://javacc.dev.java.net/files/documents/17/686/javacc-3.2.zip
 OSR: 3106
 License: BSD
+Comment: JavaCC is only used to compile a parser. javacc binaries are not included in the module's binaries.
```

Comment:
There is a header to express this. Please add this header:

```
Type: compile-time
```

That way the ant modules know how to handle this (used in VerifyLibsAndLicenses, CreateDependencies)